test/turbopack-tests-manifest.json

```diff
@@ -2567,7 +2567,8 @@
     ],
     "failed": [
       "app dir - external dependency server actions should compile server actions from node_modules in client components",
-      "app dir - external dependency should be able to opt-out 3rd party packages being bundled in server components"
+      "app dir - external dependency should be able to opt-out 3rd party packages being bundled in server components",
+      "app dir - external dependency should not apply swc optimizer transform for external packages in browser layer"
```

Comment:
I think we agreed that any new tests have to pass for turbopack with the PR they get added with 👀

Reply:
Will follow up this separately since we also need to visit the worker support in turbopack later